### PR TITLE
Docsite: update communication.rst

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
This PR:
- adds a link to the `#ansible-social` channel
- adds a paragraph about community topics
- updates the Bullhorn paragraph

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
communication.rst

### CODE REVIEW

**Comment by Andersson007** [comment]
@gundalow @cybette @samccann @GregSutcliffe @ariordan-redhat is there anything else we should add?

ready for review

File: docs/docsite/rst/community/communication.rst [hunk 1]
```diff
@@ -134,6 +137,19 @@ Meetings on chat
...
 
+Ansible Community Topics
+========================
+
+The `Ansible Community Steering Committee <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst>`_ uses the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ to asynchronously discuss with the Community and vote on Community topics in corresponding issues.
+
+Create a new issue in the `repository <https://github.com/ansible-community/community-topics/issues>`_ if you want to discuss an idea that impacts any of the following:
+
+* Ansible Community
+* Community collection best practices and requirements
```
Comment (ariordan-redhat) [inline docs/docsite/rst/community/communication.rst]: Would it be useful to include any available links for items in this list?

File: docs/docsite/rst/community/communication.rst [hunk 2]
```diff
@@ -79,11 +79,14 @@ Our IRC channels may require you to register your IRC nickname. If you receive a
...
 as described in the `libera.chat docs <https://libera.chat/guides/findingchannels>`_.
 
+.. _general_channels:
+
 General channels
 ----------------
 
 The clickable links will take you directly to the relevant Matrix room in your browser; room/channel information is also given for use in other clients:
 
+- `Community social room and posting news for the Bullhorn newsletter <https://matrix.to:/#/#social:ansible.im>`_ - ``Matrix:#social:ansible.im | IRC: #ansible-social``
```
Comment (cybette) [inline docs/docsite/rst/community/communication.rst]: space between `Matrix:` and `#social:ansible.im` for consistency

Comment (Andersson007) [reply docs/docsite/rst/community/communication.rst]: ah, thanks! fixed

File: docs/docsite/rst/community/communication.rst (hunk 1 above)
Comment (Andersson007) [reply docs/docsite/rst/community/communication.rst]: Sounds like a great idea! Added several, thanks! If you have other links on your mind, please suggest

File: docs/docsite/rst/community/communication.rst [hunk 3]
```diff
@@ -56,7 +56,7 @@ Matrix chat supports:
 * no line limits
 * images
 
-The room links in the list below will take you directly to the relevant rooms. For more information, see the community-hosted `Matrix FAQ <https://hackmd.io/@ansible-community/community-matrix-faq>`_.
+The room links in the :ref:`list below<general_channels>` will take you directly to the relevant rooms. For more information, see the community-hosted `Matrix FAQ <https://hackmd.io/@ansible-community/community-matrix-faq>`_.
```
Comment (samccann) [inline docs/docsite/rst/community/communication.rst]:
```suggestion
The room links in the :ref:`general_channels` will take you directly to the relevant rooms. For more information, see the community-hosted `Matrix FAQ <https://hackmd.io/@ansible-community/community-matrix-faq>`_.
```

File: docs/docsite/rst/community/communication.rst [hunk 4]
```diff
@@ -134,6 +137,19 @@ Meetings on chat
...
+
+The `Ansible Community Steering Committee <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst>`_ uses the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ to asynchronously discuss with the Community and vote on Community topics in corresponding issues.
+
+Create a new issue in the `repository <https://github.com/ansible-community/community-topics/issues>`_ if you want to discuss an idea that impacts any of the following:
+
+* Ansible Community
+* Community collection best practices and `requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_
+* `Community collection inclusion policy <https://github.com/ansible-collections/ansible-inclusion/blob/main/README.md>`_
+* `The Community governance <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst>`_
+* Other proposals of importance that need the Committee's attention
```
Comment (samccann) [inline docs/docsite/rst/community/communication.rst]:
```suggestion
* Other proposals of importance that need the Committee or overall Ansible community attention
```

Comment (samccann) [reply docs/docsite/rst/community/communication.rst]: I worry that limiting community-topics to just what the steering committee needs to hear would stifle conversation (like most if not all the docs-based discussion topics I've gotten great help with recently).

File: docs/docsite/rst/community/communication.rst (hunk 3 above)
Comment (Andersson007) [reply docs/docsite/rst/community/communication.rst]: Looks much better, thanks

File: docs/docsite/rst/community/communication.rst (hunk 4 above)
Comment (Andersson007) [reply docs/docsite/rst/community/communication.rst]: indeed, thanks! I'll replicate this change to the SC docs

**Comment by Andersson007** [comment]
ready for more reviews / merge

**Review by samccann — APPROVED** [review]

**Comment by Andersson007** [comment]
@ariordan-redhat @cybette @samccann thank you for reviewing and improving these changes!

**Pull request merged**